### ISSUE TITLE
Fix fall through behaviour

### ISSUE THREAD

**Issue by oz123** [oz123/coredns-netbox-plugin#2]
The "fall-through when a record doesn't exist" should not be a default/implicit behavior. See how this kind of behavior is implemented in other plugins (e.g. hosts plugin).

**Comment by oz123** [comment]
It seems this works, see example file added. It's easy to test.
Leaving this open as a reminder to follow this up  in the code for the next version.

**Comment by Azylog** [comment]
Hello,
I'm been stuck by this yesterday as my wanted behavior is : answer with record if it exists in netbox then fallback to the forward plugin to resolve more generally.
First, as netbox plugin was at the end of plugin.cfg, if the forward plugin is declared in Corefile, the netbox plugin is never used. I had to move the 'netbox:github.com/oz123/coredns-netbox-plugin' BEFORE forward in plugin.cfg.
Then, for an existent record, the netbox plugin is always answering SERVFAIL and resolution stops. So I couldn't resolve any public record.

Reading coredns documentation, I found the alternate plugin (github.com/coredns/alternate). Adding it to plugin.cfg BEFORE netbox like this : 
```
alternate:github.com/coredns/alternate
netbox:github.com/oz123/coredns-netbox-plugin
forward:forward
```
And setting Corefile like this : 
```
. {
   debug
   netbox {
      token <token>
      url <api url>
      localCacheDuration 300s
   }
   alternate SERVFAIL . 1.1.1.1
   log
   cache
}
```

It SEEMS I've been able to achieve my expected behavior : coredns can be used as internal resolver to serve netbox's records and forward to public resolver if record doesn't exists in netbox.

It still lacks lot of testing in real-life but from a PoC point of view, it's working.
